sdk-python/awaazde/constants.py

```diff
@@ -0,0 +1,12 @@
+class CommonConstants():
```

Comment:
I think this class needs to be cleaned up to only used vars or deleted altogether (I don't believe any of these are still relevant)

Reply:
@ashwini-ad Is this comment actually addressed?